packages/web/src/scss/components/Drawer/_DrawerPanel.scss

```
@@ -0,0 +1,43 @@
+@use 'sass:map';
+@use 'theme';
+
+.DrawerPanel {
+    --drawer-translate-x: 0%;
```

please prefix everything using the token `css-variable-prefix`

```suggestion
    --#{tokens.$css-variable-prefix}drawer-translate-x: 0%;
```